app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/recordaudio/RecordAudioComponent.kt

```diff
@@ -92,6 +95,11 @@ fun RecordAudioComponent(
             .fillMaxWidth()
             .fillMaxHeight()
             .background(colorsScheme().background)
+            .clickable(
+                interactionSource = remember { MutableInteractionSource() },
+                indication = null,
+                onClick = { /* Don't allow to click on elements under record area */ }
+            )
```

Comment:
suggestion: use `pointerInput` instead of `clickable` to block any touch interaction, not just clicks 
```suggestion
            .pointerInput(Unit) { /* Don't allow to click on elements under record area */ }
```